data/mods/Dark-Skies-Above/factions.json

```diff
@@ -2,38 +2,47 @@
   {
     "type": "MONSTER_FACTION",
     "name": "invader_alien",
-    "neutral": [ "small_animal", "fish", "animal", "lizard" ]
+    "base_faction": "nether",
+    "neutral": [ "alien_carnivore", "alien_herbivore", "stray", "aquatic_predator" ],
+    "by_mood": "animal"
   },
   {
     "type": "MONSTER_FACTION",
     "name": "alien_herbivore",
-    "base_faction": "small_animal"
+    "base_faction": "herbivore",
+    "neutral": [ "invader_alien", "stray" ],
+    "by_mood": [ "alien_carnivore" ]
   },
   {
     "type": "MONSTER_FACTION",
-    "name": "lizard",
-    "neutral": [ "small_animal", "invader_alien" ],
-    "by_mood": "lizard"
+    "name": "alien_carnivore",
+    "base_faction": "mutant",
+    "neutral": [ "invader_alien", "stray", "aquatic_predator" ],
+    "hate": [ "alien_herbivore" ]
   },
   {
     "type": "MONSTER_FACTION",
-    "name": "stray",
-    "//": "largely neutral to animals and things that don't live",
-    "neutral": [ "small_animal", "fish", "animal", "fox", "cat", "dog", "invader_alien" ]
+    "name": "alien_packliz",
+    "base_faction": "alien_carnivore",
+    "by_mood": "alien_carnivore"
   },
   {
     "type": "MONSTER_FACTION",
-    "name": "lurker",
-    "//": "TODO assign base faction and/or attitude"
+    "name": "stray",
+    "//": "largely neutral to animals, but will retaliate.  strictly neutral to their creators.",
+    "base_faction": "bot",
+    "neutral": [ "invader_alien", "alien_herbivore", "alien_carnivore" ]
   },
   {
     "type": "MONSTER_FACTION",
-    "name": "packliz",
-    "//": "TODO assign base faction and/or attitude"
+    "name": "lurker",
+    "base_faction": "alien_carnivore",
+    "hate": [ "fish", "aquatic_predator" ]
```

Comment:
Question regarding having "aquatic_predator" here - do you have in mind:
- something for DSA that would replace the vanilla "aquatic_predator";
- the vanilla aquatic_predator (dragonfly nymph); or
- neither of the above.
If the last, you can just remove aquatic_predator from the above, which will simplify things.

Something that goes after fish gets a bit complicated by that vanilla (particularly after the other PR) is deliberately set up not to have many things going after fish - see Venera3's discussion in #48077 of the need for this. Most things are set up to be NEUTRAL toward fish, and fish toward them. The easiest way to deal with this is to have the base_faction set to something that already goes after fish - gator seems appropriate here. The only difficulty is that you'll need to also duplicate the gator entry from #48077's data/json/monster_factions.json, so that you can modify it to be by_mood (or hate if desired) vs lurker (and lurker should be by_mood or hate vs gators). This is really a bug in the current system... there really needs to be something like a "base_faction_by_mood" that would do the same thing as base_faction, but set the base_faction pointed at to by_mood toward the new faction and vice-versa. Otherwise, you have the base faction FRIENDLY, unless it's set otherwise as with mutant and suggested for alien_carnivore above.

Reply:
The lurker is intended to be an aquatic predator in its own right that largely tries to hunt other critters in its space. However, it doesn't strictly _have_ to try and attack other aquatic predators - it might be clever enough to know doing so is dangerous or at least not worth its time. They're really there for a bit of flavor to show that the aliens are worming their way into ecological niches on Earth so easier solutions are probably better - at least for now.

Reply:
OK. I suggest removing "aquatic_predator" from the "neutral" list of invader_alien and alien_carnivore - something in that ecological spot may well try going after them, so they should be able to retaliate.

So the lurker will go after non-aquatic things? Then the best base_faction will likely be gator. That way, fish will know that it's a danger without copying over fish from vanilla+other PR - which would be problematic because of how complicated it is (see above). Then, the task is to make sure that gators will not be friendly to lurkers and vice-versa - so add "gator" to a "by_mood" for the lurker + copy over gator from vanilla+other PR and add by_mood for lurker:
```
  {
    "type": "MONSTER_FACTION",
    "name": "gator",
    "base_faction": "mutant",
    "by_mood": "lurker",
    "hate": "fish"
  },
```

Reply:
Sounds good! I'll have to check Cthorr out too :p

Reply:
I'm not sure if he ever finished the series, but it was still quite interesting.

Reply:
Wikipedia says he's still working on them, 25 years later even! Reading a few synopses, I see several parallels with the direction we're heading with Dark Skies, which is quite cool. Thanks for the recommend, should be a great source of inspiration!

Reply:
Wow. Next time I feel like reading science fiction (I used to all the time, but have gotten into other things like history and computer science papers since then) I'll have to take a look; thank you for the update!